cmd/app-proxy/main.go

```diff
@@ -36,7 +36,7 @@ func main() {
 	entrypoint.Run(run, config.New(), entrypoint.Config{ENVPrefix: ENVPrefix})
 }
 
-func run(ctx context.Context, cfg config.Config) error {
+func run(ctx context.Context, cfg config.Config, _ []string) error {
 	// Create logger
 	logger := log.NewServiceLogger(os.Stdout, cfg.DebugLog) // nolint:forbidigo
```

Comment:
There is only one binary/image/entrypoint for all Stream Service components.

You can select which components will be deployed together.

Component can be enabled by a positional argument, when running the binary.

So I have to add positional arguments to the `entrypoint` package.